pom.xml

```diff
@@ -48,71 +48,66 @@
         <java.version>1.8</java.version>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
-        <scala.version>2.11</scala.version>
-        <scala.minor.version>2.11.12</scala.minor.version>
+        <scala.minor.version>2.11</scala.minor.version>
+        <scala.version>${scala.minor.version}.12</scala.version>
 
         <!-- Dependency version properties -->
-        <junit5.version>5.6.0</junit5.version>
-        <junit.platform.suite.version>1.6.2</junit.platform.suite.version>
-        <mockito.version>3.3.3</mockito.version>
-        <failsafe.version>3.0.0-M5</failsafe.version>
-        <slf4j.api.version>1.7.25</slf4j.api.version>
+        <junit5.version>5.8.2</junit5.version>
+        <mockito.version>4.3.1</mockito.version>
+        <failsafe.version>3.0.0-M4</failsafe.version>
+        <slf4j.api.version>1.7.36</slf4j.api.version>
 
-        <koryphe.version>1.14.0</koryphe.version>
+        <koryphe.version>2.0.1</koryphe.version>
```

Comment:
Next koryphe release will be a minor release, so 2.1.0: https://github.com/gchq/koryphe/milestone/49